diff --git a/quicklisp-client/asdf.lisp b/quicklisp-client/asdf.lisp
index 855bd30..c5ae9b8 100644
--- a/quicklisp-client/asdf.lisp
+++ b/quicklisp-client/asdf.lisp
@@ -7197,7 +7197,7 @@ also \"Configuration DSL\"\) in the ASDF manual."
             (or (remove nil (getenv-absolute-directories "XDG_DATA_DIRS"))
                 (os-cond
                  ((os-windows-p) (mapcar 'get-folder-path '(:appdata :common-appdata)))
-                 (t (mapcar 'parse-unix-namestring '("/usr/local/share/" "/usr/share/")))))))
+                 (t (mapcar 'parse-unix-namestring '("/usr/lib/sdk/sbcl/share/" "/usr/local/share/" "/usr/share/")))))))
 
   (defun xdg-config-dirs (&rest more)
     "The preference-ordered set of additional base paths to search for configuration files.
diff --git a/scripts/install.sh b/scripts/install.sh
index dda09cf..4e89e19 100755
--- a/scripts/install.sh
+++ b/scripts/install.sh
@@ -2,7 +2,7 @@
 
 set -eux
 
-QLOT_SOURCE_DIR=$(realpath "$(dirname -- "${0%/*}")")
+QLOT_SOURCE_DIR="$QLOT_BASE/lib/qlot"
 
 ansi() {
   [ $# -gt 0 ] || return
@@ -11,7 +11,7 @@ ansi() {
 [ -t 1 ] || ansi() { :; }
 errmsg() { printf "%sError: %s%s\n" "$(ansi 31)" "$1" "$(ansi 0)"; }
 
-if [ "$(id -u)" -eq 0 ]; then
+if true; then
   QLOT_BASE=${QLOT_BASE:-/usr/local}
   QLOT_BIN_DIR=${QLOT_BIN_DIR:-$QLOT_BASE/bin}
 else
@@ -33,8 +33,8 @@ printf '#!/bin/sh\nexport QLOT_SETUP_FILE=%s\nexec %s/scripts/run.sh "$@"\n' \
   "$SETUP_FILE" "$QLOT_SOURCE_DIR" > "$QLOT_BIN_DIR/qlot"
 chmod 755 "$QLOT_BIN_DIR/qlot"
 
-if [ "$(id -u)" -eq 0 ]; then
-  REGISTRY_DIR=/usr/local/share/common-lisp/systems
+if true; then
+  REGISTRY_DIR=${QLOT_BASE:-/usr/local}/share/common-lisp/systems
 else
   REGISTRY_DIR="${XDG_DATA_HOME:-~/.local/share}/common-lisp/systems"
 fi
diff --git a/scripts/qlot-installer.sh b/scripts/qlot-installer.sh
index cb1e319..5250200 100755
--- a/scripts/qlot-installer.sh
+++ b/scripts/qlot-installer.sh
@@ -5,7 +5,7 @@ set -eu
 VERSION=${VERSION:-heads/master}
 QLOT_ARCHIVE=${QLOT_ARCHIVE:-https://github.com/fukamachi/qlot/archive/refs/$VERSION.tar.gz}
 
-if [ "$(id -u)" -eq 0 ]; then
+if true; then
   QLOT_BASE=${QLOT_BASE:-/usr/local}
   QLOT_HOME=${QLOT_HOME:-"$QLOT_BASE/lib/qlot"}
   QLOT_BIN_DIR=${QLOT_BIN_DIR:-"$QLOT_BASE/bin"}
@@ -127,7 +127,7 @@ echo ''
 success "Qlot v$(qlot_version) has been successfully installed under '$QLOT_HOME'."
 echo ''
 
-if [ "$(id -u)" -ne 0 ]; then
+if true; then
   echo "The executable script is located at '$QLOT_BIN_DIR/qlot'."
   echo "To make it runnable by your shell, please add '$QLOT_BIN_DIR' to '\$PATH'."
   echo ''
diff --git a/scripts/setup.sh b/scripts/setup.sh
index ef0a4d2..f139a3b 100755
--- a/scripts/setup.sh
+++ b/scripts/setup.sh
@@ -2,7 +2,7 @@
 
 set -eux
 
-QLOT_SOURCE_DIR=$(realpath "$(dirname -- "${0%/*}")")
+QLOT_SOURCE_DIR="$QLOT_BASE/lib/qlot"
 
 ansi() {
   [ $# -gt 0 ] || return
